dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/WorkflowExecuteThread.java

```diff
@@ -1662,6 +1662,18 @@ private void killAllTasks() {
                 this.addStateEvent(stateEvent);
             }
         }
+
+        if (this.stateWheelExecuteThread.clearTaskRetryList()) {
```

Comment:
Why clear retry task list here? The relationship between StateWheelExecuteThread and WorkflowExecuteThread is 1:N, so it can not clear here for some one WorkflowExecuteThread.

Reply:
I'll fix it.